default.nix

```diff
@@ -25,4 +25,5 @@
   allOverlays = import ./overlays args;
   nixpkgsArgs = { inherit config overlays system; };
   pkgs = import sources.nixpkgs nixpkgsArgs;
+  pkgs-unstable = import sources.nixpkgs-unstable nixpkgsArgs;
```

Comment:
Not sure this should still be here

Reply:
I mainly use `pkgs` for debugging haskell.nix issues, by running things like:

```
nix-build -E '(import ./. {}).pkgs.haskell-nix.tool "ghc8103" "hello" "latest"'
```

I wanted to be able to continue to do this now I have upgraded to BigSur.

Reply:
Actually, now that I look at this, it's a bit weird that `pkgs` is there too. I get that these are useful for debugging... but surely our *users* shouldn't be using these attributes?

Reply:
I wonder if we should rename them `debug` and `debug-unstable` to make that clear.

Reply:
Or add `debug.nix` with attributes useful for debugging? The problem is the dual duty of `default.nix` as the place for all useful attributes and also our de-facto external interface.